ZD6VEU/kerdesek.txt

```diff
@@ -0,0 +1,362 @@
+[
+  {
+    "kerdes": "Mi a Dependency Injection (Függőséginjektálás) célja a szoftvertervezésben?",
+    "valaszA": "Elősegíti a kód újrafelhasználhatóságát.",
+    "valaszB": "Csökkenti az alkalmazás méretét.",
+    "valaszC": "Automatizálja a tesztelést.",
+    "valaszD": "Segít csökkenteni az osztályok közötti szoros kapcsolatot.",
+    "helyes": 4,
```

Comment:
Ha válaszok ABCD suffix-el vannak, akkor a helyes is lehetett volna pl egy karakter. Pl D.